services/youtube/youtube-likes.tester.js

```diff
@@ -22,7 +22,7 @@ t.create('video vote count')
   .expectBadge({
     label: 'likes',
     message: Joi.string().regex(
-      /^([1-9][0-9]*[kMGTPEZY]?|[1-9]\.[1-9][kMGTPEZY]) 👍 ([1-9][0-9]*[kMGTPEZY]?|[1-9]\.[1-9][kMGTPEZY]) 👎$/
+      /^([1-9][0-9]*[kMGTPEZY]?|[1-9]\.[1-9][kMGTPEZY]) 👍$/
```

Comment:
Thanks for looking at the tests but as noted above we need to change the behavior a bit.

Additionally, we should instead add a new test for the scenario described in #7373. Given the specific use case of a missing `dislikeCount` field, it would probably be better for that to be a mocked test (https://github.com/badges/shields/blob/master/doc/service-tests.md#5-mocking-responses) given the likelihood for the upstream target to change/receive dislikes